Automattic/jetpack
# Boost: Add CardPage layout and move related styles

# Merge after https://github.com/Automattic/jetpack/pull/35131. Depends on changes from it.

## Proposed changes:

* Add a CardPage layout. Both Upgrade and Purchase Successful pages had a lot in common so the common parts now live in this new layout.
* Moved related styles into their respective components.
* Added header/footer to Upgrade page. The upgrade page always seemed awkward to me.

<details><summary>New upgrade page preview</summary>

<img src="https://github.com/Automattic/jetpack/assets/11799079/8f9a989e-eea1-4458-af20-a638f98ea8df" />

</details>

### Other information:


## Jetpack product discussion



Related stylesheet migration ticket - https://github.com/Automattic/jetpack/issues/35010

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup Boost;
* Check both `/wp-admin/admin.php?page=jetpack-boost#/upgrade` and `/wp-admin/admin.php?page=jetpack-boost#/purchase-successful`, and make sure everything works and looks okay.
